tests/Fixture/DoctrineCascadeRelationship/ChangesEntityRelationshipCascadePersist.php

```diff
@@ -73,7 +73,7 @@ public function setUpCascadePersistMetadata(): void
      */
     public static function provideCascadeRelationshipsCombinations(): iterable
     {
-        if (!\getenv('DATABASE_URL')) {
+        if (!\getenv('DATABASE_URL') || !self::$methodName) {
```

Comment:
this fixes a small quirks where we couldn't run tests directly with `vendor/bin/phpunit`

it fixes a small annoyance for newcomers